src/app/api/movies/[movieId]/screenings/route.ts

```diff
@@ -0,0 +1,28 @@
+import { db } from '@/utils/db';
+import Screening from '@/models/screening.model';
+import screeningModel from '@/models/screening.model';
+import { NextResponse } from 'next/server';
+import { useParams } from 'next/navigation';
+db();
+
+export async function GET(request: any, { params }: any) {
```

Comment:
When you use the `any` type, you're basically telling Typescript do not do anything, and you're not getting any benefits from using typescript. I always recommend against using (or even allowing) `any` in your code.